app/diagnostics.go

```diff
@@ -51,6 +51,7 @@ const (
 	TRACK_PERMISSIONS_GENERAL       = "permissions_general"
 	TRACK_PERMISSIONS_SYSTEM_SCHEME = "permissions_system_scheme"
 	TRACK_PERMISSIONS_TEAM_SCHEMES  = "permissions_team_schemes"
+	TRACK_EXTERNAL_SERVICES         = "external_services"
```

Comment:
@mgdelacroix I would lean towards tracking this metric with `TRACK_ELASTICSEARCH = elasticsearch` - since we might collect other info in future if valuable, like the cloud provider elasticsearch was deployed on.

This is also more consistent with other metrics like plugins and permissions.

1/5

Reply:
cc @mgdelacroix happy to discuss if you have any questions on the proposal. I'm also 1/5 on the proposal so it's up for feedback

Reply:
Sorry @jasonblais for overlooking this. I'm happy with a per-service approach instead of having a "bucket" for external services. I will prepare a PR with the necessary changes.